Makefile

```diff
@@ -378,7 +378,7 @@ scorecard-tests: operator-sdk
 .PHONY: container
 container: GOOS = linux
 container: manager
-	docker build --load -t ${IMG} .
+	docker build --build-arg TARGETARCH=amd64 --load -t ${IMG} .
```

Comment:
Why are you changing these targets? Docker should build for the host architecture by default.

Reply:
sorry  i test my local env, i will revert this